xcm/src/v0/traits.rs

```diff
@@ -67,3 +67,21 @@ impl SendXcm for () {
 		Err(Error::Unimplemented)
 	}
 }
+
+pub trait ExecuteHrmp {
```

Comment:
I am not entirely sure about naming of this. This trait seems to be focused on the channel management of HRMP. Therefore, maybe we should name it such that it reflects that? A strawman: `HrmpChannelManagementHooks`?

Reply:
accepted the strawman for now